contracts/CompoundHandler.sol

```diff
@@ -16,13 +17,16 @@ contract CompoundHandler is Handler, ReentrancyGuard {
 
     mapping(address => address) public cTokens; //cTokens;
 
+    address public compAddress;
+
     event NewMappingcToken(
         address indexed token,
         address indexed mappingcToken
     );
 
-    constructor(address _dTokenController) public {
+    constructor(address _dTokenController, address _compAddress) public {
```

Comment:
Should compound handler has an initialize() function? As it will be deployed and initialize by a proxy.

Reply:
Yes, you are right.